specs/wp-post-editor-spec.js

```diff
@@ -531,6 +531,11 @@ describe( `[${ host }] Editor: Posts (${ screenSize })`, function() {
 					'The newly scheduled post is not showing in the editor as scheduled'
 				);
 			} );
+
+			after( async function() {
```

Comment:
You probably need the _after_ here (just use a _step_) since this will run after any step that fails, so if it fails logging on then it will still try to delete the post (and also fail)